### PR TITLE
Get bindings host from pulp-smash

### PR DESCRIPTION
**Pull request by fabricio-aguiar**
https://pulp.plan.io/issues/6339
closes #6339

### CODE REVIEW

File: pulp_ansible/tests/functional/utils.py [hunk 1]
```diff
@@ -32,6 +32,7 @@
 
 
 configuration = Configuration()
+configuration.host = config.get_config().get_base_url()
 configuration.username = "admin"
```
Comment (bmbouter) [inline pulp_ansible/tests/functional/utils.py]: I see the pulp smash config also contains the username and password can this PR also have those configured from the settings too?

If we do that, with multiple configuration points coming from the smash config what about making a new function that will effectively return the fully configured `Configuration()` object that all bindings could be instantiated with?

What do you think?

Comment (fabricio-aguiar) [reply pulp_ansible/tests/functional/utils.py]: sounds good, I will do that

Comment (bmbouter) [reply pulp_ansible/tests/functional/utils.py]: @fabricio-aguiar great, I can re-review if you ping me when it's ready.

Comment (fabricio-aguiar) [reply pulp_ansible/tests/functional/utils.py]: done!

**Comment by bmbouter** [comment]
Once the pulp-smash release happens today this will be unblocked to merge. The only thing I want to request is a pulp_ansible specific issue in Redmine and have the changelog refer to that. Would you be willing to do that?

**Comment by bmbouter** [comment]
Well that release just happened :)  https://github.com/pulp/pulp_ansible/pull/300#issuecomment-603436915

**Comment by fabricio-aguiar** [comment]
trying to close and reopen to solve travis bug

**Pull request merged**